.gitignore

```diff
@@ -1,4 +1,239 @@
 
-public/
 .cache/
-node_modules/
\ No newline at end of file
+
```

Comment:
Do not revert previous changes.

Reply:
Well I didn't revert them, I guess, just reorganize them:

Line 82 (node_modules)
```
# Dependency directories
node_modules/
```

Line 126 (public)
```
# Uncomment the public line if your project uses Gatsby
# https://nextjs.org/blog/next-9-1#public-directory-support
# https://create-react-app.dev/docs/using-the-public-folder/#docsNav
public
```

But if you prefer I can move them to the top again 😄

Reply:
No, please revert this file to this commit: 1a8b0ba178fd9112b719e8cda12ea0e11897e9a3